src/app/admin/createCollectionCentre/CreateCollectionCentreForm.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from "react";
```

Comment:
"use client" here since react hook is used?

Reply:
You are right